gnomad/resources/config.py

```diff
@@ -13,17 +14,28 @@ class GnomadPublicResourceSource(Enum):
     AZURE_OPEN_DATASETS = "Azure Open Datasets"
 
 
-DEFAULT_GNOMAD_PUBLIC_RESOURCE_SOURCE = (
-    GnomadPublicResourceSource.GOOGLE_CLOUD_PUBLIC_DATASETS
-)
+def get_default_public_resource_source() -> Union[GnomadPublicResourceSource, str]:
+    """
+    Get the default source for public gnomAD resources.
```

Comment:
```suggestion
    Get the default source for public gnomAD resources.
    
    .. note::
    
        Default is pulled from the `GNOMAD_DEFAULT_PUBLIC_RESOURCE_SOURCE` environment variable if it exists. Otherwise `GOOGLE_CLOUD_PUBLIC_DATASETS` is used.
```